src/imitation/util/logger.py

```diff
@@ -108,10 +127,12 @@ def accumulate_means(self, subdir: types.AnyPath) -> Generator[None, None, None]
         if self.current_logger is not None:
             raise RuntimeError("Nested `accumulate_means` context")
 
+        name = types.path_to_str(name)
```

Comment:
This doesn't really check that name is a path if the input type is a string. Why aren't we using Pathlib?

Reply:
Good point switched name to a string and opened an issue regarding path lib.